.github/workflows/website.yml

```diff
@@ -1,15 +1,17 @@
 name: Website
 on:
   push:
-    branches: gh-pages
+    branches:
+      - gh-pages
+      - main
```

Comment:
we can also add the "master" branch here and condition the build yaml on the ref being main or master.